Buildings/Fluid/CHPs/OrganicRankine/Cycle.mo

```diff
@@ -0,0 +1,494 @@
+within Buildings.Fluid.CHPs.OrganicRankine;
+model Cycle "Organic Rankine cycle as a bottoming cycle"
+
+  extends Buildings.Fluid.Interfaces.FourPortHeatMassExchanger(
+    final m1_flow_nominal = mHot_flow_nominal,
+    final dp1_nominal = dpHot_nominal,
+    final m2_flow_nominal = mCol_flow_nominal,
+    final dp2_nominal = dpCol_nominal,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    T1_start = max(pro.T)*2/3 + min(pro.T)*1/3,
+    T2_start = max(pro.T)*1/10 + min(pro.T)*9/10,
+    redeclare final Buildings.Fluid.MixingVolumes.MixingVolume vol2(
+      V=m2_flow_nominal*tau2/rho2_nominal,
+      nPorts=2,
+      final prescribedHeatFlowRate=true),
+    final vol1(
+      final prescribedHeatFlowRate=true));
+
+  replaceable parameter Buildings.Fluid.CHPs.OrganicRankine.Data.Generic pro
+    constrainedby Buildings.Fluid.CHPs.OrganicRankine.Data.Generic
+    "Property records of the working fluid"
+    annotation(choicesAllMatching = true,Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.MassFlowRate mHot_flow_nominal
+    "Nominal mass flow rate of the evaporator fluid"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.PressureDifference dpHot_nominal(
+    displayUnit="Pa") = 0
```

Comment:
This should not have a default value of zero. It should be left unassigned so users are forced to enter a value.

Reply:
Fixed in 379c011.